### PR TITLE
CLDC-1567-footer-redesign

### PR DESCRIPTION
**Pull request by natdeanlewissoftwire**
tested locally
minimal risk
no docs required

ticket: https://digital.dclg.gov.uk/jira/browse/CLDC-1567

image

### CODE REVIEW

File: app/frontend/styles/_feedback.scss [hunk 1]
```diff
@@ -0,0 +1,122 @@
+.gem-c-feedback {
```
Comment (natdeanlewissoftwire) [inline app/frontend/styles/_feedback.scss:1]: this file contains all relevant classes from https://github.com/alphagov/govuk_publishing_components/blob/main/app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss, unaltered where possible to make any future work using that repository easier

File: app/views/layouts/_feedback.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div class="govuk-width-container">
+  <div class="gem-c-feedback">
```
Comment (natdeanlewissoftwire) [inline app/views/layouts/_feedback.html.erb:2]: these class names are copied from the `govuk_publishing_components` - I'm not actually sure why they're called gem-c-... but thought I'd use the same names for clarity

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -130,7 +130,7 @@
         <%= content_for?(:content) ? yield(:content) : yield %>
       </main>
     </div>
-
+    <%= render partial: "layouts/feedback" %>
     <%= render partial: "layouts/footer" %>
```
Comment (natdeanlewissoftwire) [inline app/views/layouts/application.html.erb:134]: the designs (https://www.figma.com/file/zeKXCOjR1DuXXIXpWHaDwS/CORE-Dev-Handover?node-id=1595%3A39150&t=3o7bIOsRN7toymMV-0) seem to remove the source code and cookies links from the footer but that wasn't mentioned in the ticket so I assume that's just a mistake and have not made that change

**Review by kosiakkatrina — APPROVED** [review]
Looks good!

**Pull request merged**